docs-source/content/userguide/managing-domains/model-in-image/new-dynamicupdate-condition.md

```diff
@@ -0,0 +1,89 @@
+#### Checking online update status
```

Comment:
Reviewers:  This is an early WIP draft .md doc. The contents will be revised and moved into the main documentation "soon".

Reply:
merged into main md